atd-etl/cris_import/cris_import.py

```diff
@@ -612,7 +612,23 @@ def align_records(map_state):
             input_column_names = util.get_input_column_names(pg, map_state["import_schema"], table, target_columns)
 
             # iterate over each imported record and determine correct action
+            should_skip_update = False
+
             for source in imported_records:
+                # Check unique key columns to make sure they all have a value
```

Comment:
We discussed specifically looking at people tables, but this should protect us from updating records in other tables when we don't have all the info we need to make sure we are updating the correct record.